### PR TITLE
[gcb-sass] Change default post/preamble to eslint

### PR DESCRIPTION
**Pull request by sadsa**
By default the SassTask config is current set to ts-lint, however, as I understand the default linter has been update to @rushstack/eslint-config. Is there any reason this hasn't been updated?

### CODE REVIEW

File: core-build/gulp-core-build-sass/src/SassTask.ts [hunk 1]
```diff
@@ -94,8 +94,8 @@ export class SassTask extends GulpTask<ISassTaskConfig> {
     super(
       'sass',
       {
-        preamble: '/* tslint:disable */',
-        postamble: '/* tslint:enable */',
+        preamble: '/* eslint-disable */',
```
Comment (octogonz) [inline core-build/gulp-core-build-sass/src/SassTask.ts:97]: Would it make sense to emit both comments, to be compatible with both ESLint and TSLint? 

I still encounter a lot of projects that have not migrated to ESLint yet.

Comment (sadsa) [reply core-build/gulp-core-build-sass/src/SassTask.ts:97]: I think for those wanting to use this package directly, i.e. without the depending on the @rushstack build tools, emitting both comments would be helpful. That way they can have their own implementation of either TSLint or ESLint and still be able to use this gulp task without additional configuration.

Comment (octogonz) [reply core-build/gulp-core-build-sass/src/SassTask.ts:97]: @sadsa Could you update your PR with this fix?

**Comment by iclanton** [comment]
Looks like this has stagnated and hasn't since been requested, so closing this PR.

**Pull request closed without merging**